app/javascript/components/organisms/StatusUpdate.js

```diff
@@ -6,7 +6,7 @@ import StatusDropdown from '../molecules/StatusDropdown'
 
 const StatusUpdate = ({ status, comment, date, onStatusDropdownChange, onAddCommentClick, statusHistoryAnchor }) => {
   return (
-    <div className='status-update'>
+    <div className={`status-update expand-wide is-${status.toLowerCase()}`}>
```

Comment:
Waiting on @slowbot on whether to add `is-{status}` on the whole component div as part of the requirements, or just the gray (based on slack convo), if needed, can keep, if not needed, just remove that one class.